cypress/support/pages/cart.pageObject.js

```diff
@@ -0,0 +1,32 @@
+import PageObject from '../PageObject';
+
+class HomeAndCataloguePageObject extends PageObject {
```

Comment:
The class name 'HomeAndCataloguePageObject' does not match the file name 'cart.pageObject.js'. The class name should represent what the class does and it should match with the file name.

Reply:
done